Day 2 - Part 2.py

```
@@ -1,29 +1,49 @@
 import copy
-def loopThrough(start: int, end: int):
-  x=0
-  listofopcodes = copy.deepcopy(opcodes)
-  
-  listofopcodes[1] = start
-  listofopcodes[2] = end
-  while True:
-    try:
-      if listofopcodes[x] == 1:
-        listofopcodes[listofopcodes[x+3]] = listofopcodes[listofopcodes[x+1]] + listofopcodes[listofopcodes[x+2]]
-      elif listofopcodes[x] == 2:
-        listofopcodes[listofopcodes[x+3]] = listofopcodes[listofopcodes[x+1]] * listofopcodes[listofopcodes[x+2]]
-      if listofopcodes[0] == 19690720:
-        print(str(start) + " " + str(end))
-    except:
-      break

```

Note: I removed the try/except.
What you are doing here is eating the exception. I assume the exception you are getting is an invalid index exception, since you are not terminating you while loop at the end of the list. 

We want the exceptions to " bubble up" to main, so the program stops when something bad happens. If you are getting a out-of-range index exception, that means your program has a bug and it needs to be fixed. We don't want to suppress those. In your case, it will silently return for any number of reasons - out of memory, etc. 

Modify the loop to terminate the correctly when it reaches the end of the list.